browses.md

```diff
@@ -54,6 +54,27 @@ Error codes:
 
 **404** if the browse index doesn't exist
 
+## Browse indexes configured for metadata linking
+### Get a single index configured to render given field as a link
+**/api/discover/browses/search/byField?field=<:field>**
+
+Example: <http://dspace7.4science.it/dspace-spring-rest/#http://dspace7.4science.it/dspace-spring-rest/api/discover/browses/search/byField/dc.contributor.author>
+
+Provide detailed information about a specific browse index and access to the list of items and entries in the index. The JSON response document is as the previous section.
+
+The supported parameters are:
+* field: The metadata field name, eg. dc.contributor.author
+
+Error codes:
+**404** if the browse index is not a metadata browse or the browse index doesn't exist at all
+
+### Get all browse indexes configured to render metadata links
+**/api/discover/browses/search/allLinked**
+
+Example: <http://dspace7.4science.it/dspace-spring-rest/#http://dspace7.4science.it/dspace-spring-rest/api/discover/browses/search/allLinked>
```

Comment:
can you please update the url to the current demo?

Reply:
I've removed this endpoint and doc